packages/shared/src/components/CombatReport/CombatReplay/index.tsx

```diff
@@ -222,18 +240,25 @@ export function CombatReplay() {
       <div className={`flex flex-col flex-1 rounded-box bg-base-200 overflow-hidden relative`}>
         <div className="w-full h-full absolute" ref={initializeReplayContainerRef}>
           {replayContainerRef ? (
-            <Stage
-              width={replayContainerRef.clientWidth}
-              height={replayContainerRef.clientHeight}
-              options={{
-                antialias: true,
-                autoDensity: true,
-                backgroundAlpha: 0,
-              }}
-              onMount={setPixiApp}
-              onUnmount={() => {
-                setPixiApp(null);
-              }}
+            <ContextBridge
```

Comment:
interesting... what's special about CombatReplay that makes this necessary?

Reply:
Contexts aren't passed into custom renderers implicitly I guess -- more at https://reactpixi.org/context-bridge